config/routes.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "user#index"
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
+  resources :user, only: [:index, :show] do
```

Comment:
- [ ] Your route here is named `user` & `post` but you would have multiple users and posts so it should be `users & posts` :+1: :sparkles:

Reply:
Hi @Efiamotu-1 

It appears that I'll be attempting to reference the UsersController while in my case I generated a UserController and a PostController